entry/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "entry",
+  "version": "1.0.0",
+  "ohos": {
+    "org": "huawei",
+    "buildTool": "hvigor",
+    "directoryLevel": "module"
+  },
+  "description": "example description",
+  "repository": {},
+  "license": "ISC",
+  "dependencies": {
+    "@ohos/materialtextfield": "file:../materialtextfield",

```

Comment:
remove textfield library as it is not used

Reply:
Removed textfield library